bridge/bridge.go

```diff
@@ -246,6 +246,46 @@ func (b *Bridge) add(containerId string, quiet bool) {
 	}
 }
 
+func mapTags(metadata map[string]string, container *dockerapi.Container) []string {
+	// take the tags, parse them and replace those
+	// inside {{ }} with the path in the container description
+	// if found, else replace by empty string
+	var tags = make([]string, 0)
+	tag_re := regexp.MustCompile("([^,]+)")
```

Comment:
Would it be possible to implement this with text.Template? I would rather not introduce regexp if it isn't required.